powershell/public/Invoke-Maester.ps1

```diff
@@ -154,6 +157,9 @@ function Invoke-Maester {
         # Run the tests in non-interactive mode. This will prevent the test results from being opened in the default browser.
         [switch] $NonInteractive,
 
+        # This will suppress all pretty messages.
+        [switch] $Silent,
```

Comment:
Could this be something that we utilize the existing **Verbosity** parameter for?